bpf/lib/common.h

```diff
@@ -1086,6 +1094,14 @@ struct lb4_service {
 	 * slots under quarantine (otherwise zero).
 	 */
 	__u16 qcount;
+#ifdef PER_SVC_LB
+    /* Load balancer algorithm
+     * 1 - random
+     * 2 - maglev
+     */
+    __u8 lb_alg;
```

Comment:
Couldn't we start at 0 == random?

Reply:
Ah, nevermind because you are reusing LB_SELECTION_* ok, that's fine then.

Reply:
yes, that's correct